apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -540,6 +543,15 @@ export const OrganizationFooter: React.FC<
         />
       )
       break
+    case 'rettindagaesla-fotlun':
```

Comment:
Could you look into using the <Footer /> component instead of having a custom footer component? You can look at how the Samgöngustofa footer is implemented to get an idea of what I mean